protopipe/aux/example_config_files/protopipe/performance.yaml

```diff
@@ -1,11 +1,13 @@
+
 general:
  # Directory with input data file
- # [...] = your analysis local full path OUTSIDE the Vagrant box
+ # [...] = your analysis local path
+ # Please refer to directory structure shown at Lugano
  indir: '[...]/data/DL2'
  # Template name for input file
- template_input_file: 'DL2_{}_{}_merged.h5' # filled with mode and particle type
+ template_input_file: 'dl2_{}_{}_merged.h5' # filled with mode and particle type
  # Directory for output files
- outdir: '[...]/data/DL3'
+ outdir: '[...]/performance'
  # Output table name
  output_table_name: 'table_best_cutoff'
 
```

Comment:
Please, revert all the changes above this line since I have updated the interface with a new release (repository [here](https://github.com/HealthyPear/protopipe-grid-interface), new documentation to be loaded with protopipe 0.3.0)